lib/cloudfunc.js

```diff
@@ -138,6 +138,21 @@
             
             return pathHTML;
         }
+
+        /*
+         * OSC_CUSTOM_CODE change the date format from 'DD.MM.YYYY' to 'MM/DD/YYYY' format
+         * @param dotDate - The date in format 'DD.MM.YYYY'
+         * @return string - The date in format 'MM/DD/YYYY'
+         *                  The date in format '          ' if input is null.
+         */
+        function buildPrettyDate(dotDate) {
+            var prettyDate = '           ';
+            if (dotDate != null) {
+                var dmy = dotDate.split('.');
+                prettyDate = dmy[1] + '/' + dmy[0] + '/' + dmy[2];
```

Comment:
Maybe instead of this you should do:

```js
if (dotDate != null) {
  prettyDate = dotDate.replace(/(\d\d).(\d\d).(\d{4})/, "$2/$1/$3");
}
```

If the regex doesn't match the value returned is just dotDate. Just in case for whatever reason dotDate was a non null string that didn't follow the explicit format "DD.MM.YYYY"